nexus/src/app/instance.rs

```diff
@@ -563,9 +563,12 @@ impl super::Nexus {
         // outright fails, this operation fails. If the operation nominally
         // succeeds but nothing was updated, this action is outdated and the
         // caller should not proceed with migration.
-        let (updated, _) = match instance_put_result {
+        let (updated, _, _) = match instance_put_result {
             Ok(state) => {
                 self.write_returned_instance_state(&instance_id, state).await?
+
+                // TODO(eliza): this is where we would also want to insert to the
```

Comment:
Is this comment accurate? This looks like it's already happening in the migration saga proper (which is where I'd expect it to happen).

Reply:
yeah, this is out of date, my bad --- `write_returned_instance_state` now calls
into `instance_and_vmm_update_runtime`, which does write to the `migration` table.